### DISCUSSION TITLE
AVIF documentation

### DISCUSSION THREAD

**Discussion by kkmuffme** [ImageMagick/ImageMagick#4377] [Development] [answered]
Could we get a page like https://imagemagick.org/script/webp.php for AVIF?

I don't really know where those arguments are from, otherwise I would create the page myself.

**Comment by urban-warrior** [comment, marked as answer]
Not much to say about AVIF.  It's part of an HEIC container.  The HEIC coder supports a few settings described @ https://imagemagick.org/script/defines.php.  Refer to the [libheif](https://github.com/strukturag/libheif) documentation to determine the particulars of how that delegate library supports the AVIF format.
